fiftyone/operators/types.py

```diff
@@ -956,13 +972,15 @@ def __init__(self, **kwargs):
         self.params = kwargs.get("params", None)
 
     def to_json(self):
-        return {
-            **super().to_json(),
-            "href": self.href,
-            "operator": self.operator,
-            "params": self.params,
-            "prompt": self.prompt,
-        }
+        return _convert_callables_to_operator_uris(
```

Comment:
Looks like this would support `operator` to be a method of the operator class. It may be worth updating docs at line 962 of this file